packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.scss

```diff
@@ -53,14 +53,9 @@
         }
     }
 
-    &__success-footer {
-        display: flex;
-        gap: 0.8rem;
-
-        @include mobile {
-            flex-direction: column;
-            padding-block: 1.6rem;
-            width: 100%;
+    &__success-buttons {
+        &--desktop {
```

Comment:
v.minor: typically its better to keep modifiers at top level (so the modifier does not need to be repeated for every block which depends on it)